src/sim/SimTrackView.hh

```diff
@@ -35,6 +35,12 @@ class SimTrackView
     // Initialize the sim state
     inline CELER_FUNCTION SimTrackView& operator=(const Initializer_t& other);
```

Comment:
The initializer should reset the step count, right?

Reply:
Yep, it resets it.